hs/src/Reach/Simulator/Core.hs

```diff
@@ -703,6 +703,8 @@ instance Interp DLExpr where
       setGlobal $ g {e_nctc = ctcId}
       -- XXX make an interact point with dr
       return $ V_Contract $ fromIntegral ctcId
+    DLE_ContractFromAddress _at _addr -> do
+      return $ V_Data "None" V_Null
```

Comment:
You should make a comment here that a better impl is to make it a choice OR consult the table of apps